docs/gpu.md

```diff
@@ -152,10 +152,32 @@ The only scenario where this would be required is if `$LD_LIBRARY_PATH` is modif
 
 ### Testing the GPU support {: #gpu_cuda_testing }
 
-The quickest way to test if software installations included in  EESSI can access and use your GPU is to run the
+The quickest way to test if software installations included in EESSI can access and use your GPU is to run the
 `deviceQuery` executable that is part of the `CUDA-Samples` module:
-```
+```{ .bash .copy }
 module load CUDA-Samples
 deviceQuery
 ```
-If both are successful, you should see information about your GPU printed to your terminal.
+If both are successful, you should see information about your GPU printed to your terminal, for example:
+
+```
+$ deviceQuery
+deviceQuery Starting...
+
+ CUDA Device Query (Runtime API) version (CUDART static linking)
+
+Detected 1 CUDA Capable device(s)
+
+Device 0: "NVIDIA A2"
+  CUDA Driver Version / Runtime Version          12.2 / 12.1
+  CUDA Capability Major/Minor version number:    8.6
+...
+```
+
+If the `deviceQuery` command can not access your GPU, you will see an error message like:
```

Comment:
This shouldn't actually happen though, because of the Lmod guards the only scenario I can see where you would reach this is where you are using a container and the system drivers are too old

Reply:
I triggered it by cleaning out the `host_injections` directory after loading the module.

I agree it's very unlikely that it happens, but we should mention it in the docs regardless, if only to let people easily find this page when searching for error messages.

Reply:
My concern here is that the placement here makes it seem like it not working is likely, but reaching this message is actually very unlikely

Reply:
Maybe a little box saying `What does it look like if the command fails?`